conda-forge.yml

```diff
@@ -3,3 +3,7 @@ azure:
   settings_win:
     pool:
       vmImage: windows-2022
+build_platform:
```

Comment:
Don't touch this file, please revert the changes here.

Reply:
Ok, I will. How should I add `osm-arm64` build?